cmake/linkerFlagsTargets.cmake

```diff
@@ -1,6 +1,4 @@
-if(MSVC)
-elseif(CMAKE_CXX_COMPILER_ID MATCHES "^Apple")
-elseif(CMAKE_CXX_COMPILER_ID STREQUAL "GNU" OR CMAKE_CXX_COMPILER_ID STREQUAL "Clang")  # TODO: change based on linker used, not compiler
+if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU" OR CMAKE_CXX_COMPILER_ID STREQUAL "Clang")  # TODO: change based on linker used, not compiler
```

Comment:
I left them here so that there is no need to lookup how to check for different compilers.